apps/hash-ai-worker-py/app/workflows.py

```diff
@@ -0,0 +1,17 @@
+from datetime import timedelta
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
+    from .activities import complete
+
+
+@workflow.defn
+class DemoWorkflow:
```

Comment:
Not suggesting we should change this, but having them both be the same workflow (or rather them having the same WorkflowID, even if our workers are subscribing to different task queues) is slightly concerning to me, because it might mislead people's intuition in ways we don't want to right now. 

When we have some form of statically defining workflows (codegen the signatures or whatever) then I think it would be a great example to have the same workflow in both, but until then, I think it'd be too easy to change the signature of one, or something else without realising the implications.. I think I'd just opt for distinct workflow names at this time, and look to have a cohesive demo when we have a codegen thingy